docs/UserGuide.md

```diff
@@ -30,6 +32,22 @@ This section serves to help the user better understand the terminologies used in
 
 # Features
 
+## Input of data: `input`
+
+Stores the data to be processed by modules.
+
+Format: `input <data>`
+
+> 💡 `data` will be treated as text. Also, note that the program will take whatever characters after the first space character after the `input` command as `data`.  
```

Comment:
Awkward phrasing. Consider changing to "Note that the program will consider all characters as `data` following the first space character in the `input` command".